src/components/FileUploader/FileUploader-story.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { FileUploader, FileUploaderButton } from 'carbon-components-react';
+import { FileUploaderButton } from 'carbon-components-react';
+import { FileUploader } from '../../index';
 import { withReadme } from 'storybook-readme';
 import readme from './README.md';
 
+const tooltipProps = {
+  text: 'only .jpg files at 500mb or less',
+  iconName: 'info',
+  menuOffset: {
+    top: -128,
```

Comment:
This is to accommodate the button height beneath?

Reply:
yup. it places it here:
<img width="404" alt="screen shot 2018-03-13 at 10 58 50 am" src="https://user-images.githubusercontent.com/11317322/37354824-e81b8cf6-26b8-11e8-9f53-a490af12ad70.png">